### PR TITLE
Link pre-v2.0-3e65111 docs to corresponding rustdocs and rename version from pre-2.0

### PR DESCRIPTION
**Pull request by JoshOrndorff**
This PR updates all links from the `pre-2.0` devhub content to the corresponding (`pre-v2.0-3e65111`) version of the rustdocs.

~~**DO NOT MERGE YET**~~

The change to Substrate's CI went in [here](https://github.com/paritytech/substrate/pull/4836)
The tag was pushed [here](https://github.com/paritytech/substrate/releases/tag/pre-v2.0-3e65111)
The built rustdocs have shown up [here](https://substrate.dev/rustdocs/pre-v2.0-3e65111/sc_service/index.html)

### CODE REVIEW

**Comment by JoshOrndorff** [comment]
These docs are now built and this PR can safely be merged. Reviews appreciated.

File: docs/tutorials/start-a-private-network/compile.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-Substrate does not (yet) offer binary installation packages, so it must be compiled from source, which can be a time-consuming process. It is wise to start the compilation and then read the next few pages of the tutorial while your node is building. This tutorial will use the Substrate node template.
+Substrate does not (yet) offer binary installation packages, so it must be compiled from source, which can be a time-consuming process. It is wise to start the compilation and then read the next few pages of the tutorial while your node is building. This tutorial will use the Substrate node template (version `pre-2.0-3e65111`).
 
 ```bash
 # Install rust prerequisites
 curl https://getsubstrate.io -sSf | bash -s -- --fast
 
 # Download the Substrate node template
-git clone https://github.com/substrate-developer-hub/substrate-node-template.git
+git clone -b pre-v2.0-3e65111 https://github.com/substrate-developer-hub/substrate-node-template
```
Comment (riusricardo) [inline docs/tutorials/start-a-private-network/compile.md]:
```suggestion
git clone -b pre-v2.0-3e65111 --depth 1 https://github.com/substrate-developer-hub/substrate-node-template
```

Comment (riusricardo) [reply docs/tutorials/start-a-private-network/compile.md]: This change will clone the specified commit/tag. There is no need to clone all the history.

File: website/versioned_docs/version-pre-v2.0-3e65111/tutorials/creating-your-first-substrate-chain/setup.md [hunk 2]
```diff
@@ -50,13 +50,14 @@ well documented [here](overview/getting-started.md).
...
-a simple template project to help you get started building on Substrate.
+Once the prerequisites are installed, you need to set up the skeleton for our project. The Substrate Node
+Template serves as a good starting point for building on Substrate.
 
-1. Clone the Substrate Node Template
+1. Clone the Substrate Node Template (version `pre-2.0-3e6511`) and create a branch for your work.
 
     ```bash
-    git clone https://github.com/substrate-developer-hub/substrate-node-template
+    git clone -b pre-v2.0-3e65111 https://github.com/substrate-developer-hub/substrate-node-template
```
Comment (riusricardo) [inline website/versioned_docs/version-pre-v2.0-3e65111/tutorials/creating-your-first-substrate-chain/setup.md]:
```suggestion
    git clone -b pre-v2.0-3e65111 --depth 1 https://github.com/substrate-developer-hub/substrate-node-template
```

File: docs/tutorials/creating-your-first-substrate-chain/setup.md [hunk 3]
```diff
@@ -48,13 +48,14 @@ well documented [here](overview/getting-started.md).
...
-a simple template project to help you get started building on Substrate.
+Once the prerequisites are installed, you need to set up the skeleton for our project. The Substrate Node
+Template serves as a good starting point for building on Substrate.
 
-1. Clone the Substrate Node Template
+1. Clone the Substrate Node Template (version `pre-2.0-3e6511`) and create a branch for your work.
 
     ```bash
-    git clone https://github.com/substrate-developer-hub/substrate-node-template
+		git clone -b pre-v2.0-3e65111 https://github.com/substrate-developer-hub/substrate-node-template
```
Comment (riusricardo) [inline docs/tutorials/creating-your-first-substrate-chain/setup.md]:
```suggestion
		git clone -b pre-v2.0-3e65111 --depth 1 https://github.com/substrate-developer-hub/substrate-node-template
```

File: website/versioned_docs/version-pre-v2.0-3e65111/tutorials/start-a-private-network/compile.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-Substrate does not (yet) offer binary installation packages, so it must be compiled from source, which can be a time-consuming process. It is wise to start the compilation and then read the next few pages of the tutorial while your node is building. This tutorial will use the Substrate node template.
+Substrate does not (yet) offer binary installation packages, so it must be compiled from source, which can be a time-consuming process. It is wise to start the compilation and then read the next few pages of the tutorial while your node is building. This tutorial will use the Substrate node template (version `pre-2.0-3e65111`).
 
 ```bash
 # Install rust prerequisites
 curl https://getsubstrate.io -sSf | bash -s -- --fast
 
 # Download the Substrate node template
-git clone https://github.com/substrate-developer-hub/substrate-node-template.git
+git clone -b pre-v2.0-3e65111 https://github.com/substrate-developer-hub/substrate-node-template
```
Comment (riusricardo) [inline website/versioned_docs/version-pre-v2.0-3e65111/tutorials/start-a-private-network/compile.md]:
```suggestion
git clone -b pre-v2.0-3e65111 --depth 1 https://github.com/substrate-developer-hub/substrate-node-template
```

File: docs/tutorials/creating-your-first-substrate-chain/setup.md [hunk 5]
```diff
@@ -48,13 +48,14 @@ well documented [here](overview/getting-started.md).
...
+		git checkout -b my-first-substrate-chain
```
Comment (riusricardo) [inline docs/tutorials/creating-your-first-substrate-chain/setup.md]:
```suggestion
		cd substrate-node-template
		git checkout -b my-first-substrate-chain
```

Comment (riusricardo) [inline docs/tutorials/creating-your-first-substrate-chain/setup.md]: This command cannot be here. It needs to be on step **3**, after `cd substrate-node-template`

File: website/versioned_docs/version-pre-v2.0-3e65111/tutorials/creating-your-first-substrate-chain/setup.md [hunk 6]
```diff
@@ -50,13 +50,14 @@ well documented [here](overview/getting-started.md).
...
+    git checkout -b my-first-substrate-chain
```
Comment (riusricardo) [inline website/versioned_docs/version-pre-v2.0-3e65111/tutorials/creating-your-first-substrate-chain/setup.md]: Same as above. This command cannot be here. It needs to be on step **3**, after `cd substrate-node-template`

**Pull request merged**